modules/monitoring-contents-of-the-metrics-ui.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster-monitoring/examining-cluster-metrics.adoc
+
+[id="contents-of-the-metrics-ui_{context}"]
+= Contents of the Metrics UI
+
+This section shows and explains the contents of the Metrics UI, a Web interface to Prometheus.
+
+The *Metrics* page is located in *Monitoring* -> *Metrics* of the {product-title} web console.
```

Comment:
I haven't written any UI docs, so I don't know what we're calling these things.

Is it the Monitoring -> Metrics section? Pane? Panel?

The **Metrics** page is located in **Monitoring** -> **Metrics** section of the {product-title} web console.

Reply:
A valid suggestion from the POV of English. However, I just went through all occurrences of this and specifying this everywhere would add too much noise to the text. When a user reads this, they'll know to just click something saying **Monitoring** and then something saying **Metrics** and they won't care what it's called.
Besides, **Monitoring** -> **Metrics** here is really a navigation instruction that leads to the **Metrics** page.

Reply:
Read your later comment and reconsidered. Implemented this across the whole of monitoring docs. Thanks!